lib/rules/cli/4.1.yaml

```diff
@@ -573,6 +573,14 @@
     :source_spec: <value>
     :strategy: --strategy=<value>
     :template: --template=<value>
+:new_app_as_dc:
+  :cmd: oc new-app
+  :options:
+    :code: --code=<value>
+    :image_stream: --image-stream=<value>
+    :name: --name=<value>
+    :source_spec: <value>
+    :app_repo: <value>
```

Comment:
@yapei  Could you sort them in alphanumeric order ?

Reply:
sure, let me update